gradle/libs.versions.toml

```diff
@@ -69,7 +65,7 @@ managed-r2dbc-mssql = { module = "io.r2dbc:r2dbc-mssql", version.ref = "managed-
 managed-r2dbc-postgresql = { module = "org.postgresql:r2dbc-postgresql", version.ref = "managed-r2dbc-postgresql" }
 
 # Reactor Netty Dependencies
-managed-reactor-netty-http = { module = "io.projectreactor.netty:reactor-netty-http", version.ref = "managed-reactor-netty" }
+managed-reactor-netty-http = { module = "io.projectreactor.netty:reactor-netty-http" }
```

Comment:
Building app fails due to error 
```
Could not determine the dependencies of task ':micronaut-r2dbc-bom:publishMavenPublicationToMavenLocal'.
> Could not create task ':micronaut-r2dbc-bom:generateCatalogAsToml'.
   > A dependency constraint was added on 'io.projectreactor.netty:reactor-netty-http' without a version. This is invalid: a constraint must specify a version.
```
because this dep declaration is missing version

Reply:
And another issue is that micronaut-data needs netty-reactor-core as well, or else Postgres r2dbc tests are failing